docs/src/prototypes/chatPane/chatPaneContent.tsx

```diff
@@ -38,26 +38,35 @@ class ChatPaneContainer extends React.PureComponent<ChatPaneContainerProps> {
   }
 
   private generateChatItems(chat: ChatData): JSX.Element[] {
-    return generateChatProps(chat).map(({ itemType, ...props }, index) => {
-      const ElementType = this.getElementType(itemType)
-      const maybeAttributesForDivider =
-        itemType === ChatItemTypes.divider
-          ? {
-              role: 'heading',
-              'aria-level': 3,
+    return generateChatProps(chat).map(
+      ({ mine, gutter, content: { itemType, ...props } }, index) => {
+        const ElementType = this.getElementType(itemType)
+        const maybeAttributesForDivider =
+          itemType === ChatItemTypes.divider
+            ? {
+                role: 'heading',
+                'aria-level': 3,
+              }
+            : {}
+        return (
+          <Chat.Item
+            key={`chat-item-${index}`}
+            gutterPosition={mine ? 'end' : 'start'}
```

Comment:
Aaah got it, you have it here. Disregard my https://github.com/stardust-ui/react/pull/556/files#r240708414